apis/metal3.io/v1alpha1/baremetalhost_types.go

```diff
@@ -1,5 +1,5 @@
 /*
-
+Copyright 2021.
```

Comment:
Could we not, or is it doing to want to replace this every time we change something?

Reply:
Good question. Not sure if thats the case. Shall I remove it from all the files?

Reply:
I'd prefer that, it can only get out of date.

Reply:
Done